### PR TITLE
the clientConfiguration could now be configured inside the .atoum.php file

### PR DESCRIPTION
**Pull request by agallou**
This PR on atoum will enable extensions to be configured in
the .atoum.php : atoum/atoum#529

We now can pass the ClientConfiguration to the extension.

This will remove the boilerplate inside the test file, it will
no longer be required to pass the Client to the asserter.

### CODE REVIEW

**Comment by jubianchi** [comment]
:+1:

**Comment by agallou** [comment]
when the next version of atoum that will include the configuration mechanism will be tagged/released. We should add this version as a minimum requirement for the extension.

**Comment by jubianchi** [comment]
let's ping @Hywan and @mikaelrandy to see if we can issue a new release soon to include the new configuration mechanism ;)

**Comment by Hywan** [comment]
@jubianchi I am fine with this. Is there any new features that are almost ready?

**Comment by mikaelrandy** [comment]
@jubianchi @Hywan why break the release process (every 6 weeks) ?
FYI, the next standard release is planned for 15-01-2016.

BTW, i have no reason to be against.

**Comment by Hywan** [comment]
@mikaelrandy Not breaking. Just this is important from a marketing point of view. This extension is important and it requires new features introduced in `atoum@dev-master`, so it makes sense.

**Comment by mikaelrandy** [comment]
As @agallou say [here](https://github.com/agallou/blackfire-extension/pull/3#issuecomment-163381251), it's just a question of using tagged version ;)

Why do not continue development of the alpha/beta extension with `dev-master`, and wait for the next stable release ?

Otherwise, no objection to release a new version of atoum/atoum

**Comment by Hywan** [comment]
@mikaelrandy Correct. So, we do not break the release cycle and keep going :-).

**Comment by agallou** [comment]
if there is no tag on atoum/atoum, the extension could not be tagged with this change. 
It will no be "easy" to need both the extension and atoum on dev-master to use the latest functionalities.

**Comment by jubianchi** [comment]
i merged a bunch of PRS in atoum so the release could be interesting for xmas or even before :)

**Comment by agallou** [comment]
@jubianchi great, good news !

@jubianchi @Hywan @mikaelrandy I'ts not relevant for the PR, but what do you think about moving this repo to the atoum organization ? is it ok for you ?

**Comment by Hywan** [comment]
It is!

**Comment by jubianchi** [comment]
@agallou go move it! it's an awesome ext and totally compliant with standard extensions ;)

**Comment by mikaelrandy** [comment]
@agallou @jubianchi @Hywan on the versionning debate, there is my points : 
I have no objection to release a "special christmas atoum version" :P. I find it funny.
I remain unconvinced by the urgent need, but i can understand this extension could be a good advertising for atoum. I can also understand that Decitre is one of the main companies that uses us and we could make them a gift.

TL;DR
I have no objection, but i'm not convinced. So i let you decide what to do ;)

**Comment by agallou** [comment]
@mikaelrandy this extension is currently not used at Decitre (and wasn't developed to be used here).
